doc/diary/2016_02_01.md

```diff
@@ -6,8 +6,10 @@ turned out to be painfull and the setup was really ugly. It also did not work
 in all cases and the program had some unpredictable side effects. Something
 else was needed. The factory concept was cool though and I wanted to keep it. I
 realized there is already something that abstracts communication and decouples
-dependencies, the [gateway](/doc/concept/factory.md)! So I removed the multi
-state concept. Made the code cleaner. Made the ste concept work.
+dependencies, the
+[gateway](https://github.com/xh3b4sd/anna/blob/43ecdc8c9ce05bb41c3f66bff4dde90a0552dc59/doc/concept/gateway.md)!
+So I removed the multi state concept. Made the code cleaner. Made the ste
```

Comment:
Good catch. Not 100 percent sure. I guess it should be `state`. So did I fix it.